setup.py

```diff
@@ -1,4 +1,11 @@
+import os
```

Comment:
Better to use setuptools_scm, like we do in e.g. ``tszip``. (We don't use it in tskit because the package structure is too complicated)

https://github.com/tskit-dev/tszip/blob/main/setup.py